modules/login/login.py

```diff
@@ -17,6 +18,7 @@
 @login_blueprint.route("/")
 def index():
     """The entry point, serving different contents depending on the user's log in status."""
+    database.Database().setup()
```

Comment:
This function is not meant to be called. It's supposed to be get().

Reply:
Furthermore, where are all the other database/table calls? This line is practically meaningless.

Reply:
Database().setup() is not used anywhere in our source, how are we gonna initially set it up? that was how I was able to get it to work

Reply:
I think it should be done at the entry point (spotipyn.py) instead.